$OpenBSD: patch-src_user-classify_c,v 1.2 2013/12/17 08:20:04 ajacoutot Exp $

Add all ports and system users that have a login shell.

--- src/user-classify.c.orig	Tue Oct 15 22:25:19 2013
+++ src/user-classify.c	Tue Dec 17 09:19:06 2013
@@ -54,11 +54,24 @@ static const char *default_excludes[] = {
         "man",
         "at",
         "gdm",
-        "gnome-initial-setup"
+        "gnome-initial-setup",
+        "_couchdb",
+        "_ejabberd",
+        "_jabberd",
+        "_mediatomb",
+        "_mon",
+        "_openfire",
+        "_postgresql",
+        "_rancid",
+        "_riak",
+        "_rocrail",
+        "_sabnzbd",
+        "_sogo",
+        "_varnish"
 };
 
 #define PATH_NOLOGIN "/sbin/nologin"
-#define PATH_FALSE "/bin/false"
+#define PATH_FALSE "/usr/bin/false"
 
 static gboolean
 user_classify_is_excluded_by_heuristics (const gchar *username,
